tests/test_wav_manager.py

```diff
@@ -0,0 +1,33 @@
+import os
```

Comment:
I run into this error when I try to run this:

FileNotFoundError: [Errno 2] No such file or directory: '../../_data/list_of_stimuli.yaml'

I think you implemented the importlib.resource thing which is how you get it to work?

Reply:
Sorry for the error, I think the importlib.resources tool should get this to work - was this before this change was applied?